.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install Cruft and Poetry
-        run: pip install --no-input cruft poetry
+        run: pip install --no-input cruft poetry "click<=8.0.4"  # pinned to work around https://github.com/tiangolo/typer/issues/377, change when that is fixed
```

Comment:
For my information, why is this needed here? Isn't it line 50 that will actually install click?

Reply:
They both install it. Check this run (commit no longer exists as I amended it, but it didn't have the change on this line): https://github.com/radix-ai/poetry-cookiecutter/runs/5735889706. If you check the "Install Cruft and Poetry", you can see that `click` gets installed in that step as well, with a version that has the bug.